rest/data/patch.go

```diff
@@ -21,7 +21,7 @@ import (
 	"github.com/pkg/errors"
 )
 
-func validatePatchID(patchId string) error {
+func ValidatePatchID(patchId string) error {
```

Comment:
Why was this function made public?

Reply:
It is now being used in graphql package in ChildVersions since the data function that it called (that is now removed) used this function.